### ISSUE TITLE
Login folder missing?

### ISSUE THREAD

**Issue by alexcarisey** [drdavidpier/CellKulture#1]
I’m stuck shortly after the installation process. I can see the login page but cannot go any further. I get a ‘Not Found’ error: The requested URL /app/login/validate was not found on this server. I played with it for a while but I’m stuck and I’m thinking that maybe it’s simply that the ‘login’ folder is gone and the validate script has been moved somewhere else or been renamed in the latest distribution? Is it the case or am I missing something? Thanks!

PS: I’ve installed the database (through phpMyAdmin) and edited database.php and config.php to match my environment (local server on a OS X 10.10, Apache 2.4.9, php 4.3.12, mysql 5.6.23).

**Comment by drdavidpier** [comment]
Looks like a problem with the url thats coded in to the login view. If you add in index.php?/ before the rest of the link this will work as a hack.

Looks like a problem with removing the index.php? that needs mod_rewrite enabled (it should be as its in the .htcaccess file) and works on my system. Not sure where the issue is at the moment

**Comment by alexcarisey** [comment]
Tried a bunch of things but still stuck so it's probably because of my limited knowledge on server tricks. Thanks anyway!

**Comment by st0w** [comment]
The problem is a missing .htaccess in the app directory.  In both the downloaded zip and the Git repo itself, the file doesn't exist.  If you create a new file named '.htaccess' (no quotes) and place the contents below, it will function as expected:

 RewriteEngine On
 RewriteBase /app/
 RewriteCond %{REQUEST_FILENAME} !-f
 RewriteCond %{REQUEST_FILENAME} !-d
 RewriteRule ^(.*)$ index.php?/$1 [L]

**Comment by alexcarisey** [comment]
Thanks stew for the update. I've made the suggested addition of the .htaccess file but still no luck. I always get the same 404: The requested URL /app/index.php was not found on this server.
I've checked the config.php and database.php files and everything seems ok. They do contain the relevant path and credentials.
I'm not sure what I can try next but thank you for your help in any case, much appreciated.
